examples/_dev/src/components/Account.tsx

```diff
@@ -18,8 +18,9 @@ export const Account = () => {
   })
   const ensAvatar = useEnsAvatar({
     addressOrName: account?.address,
+    chainId: 1,
```

Comment:
Should this just be the default for ENS queries? I feel like you more often want the ENS data on mainnet. It's rare that I see someone with an e.g. rinkeby ENS set up properly.

Reply:
Could make sense. RainbowKit might be heading [in that direction](https://github.com/rainbow-me/rainbowkit/pull/496). Should think about if there are any arguments against this and address in a separate PR.